engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -13,29 +13,26 @@ <h2 class="fl" translate>Content Host Registration</h2>
     </div>
   </header>
 
-  <p translate>To register a content host to this server, run the following commands within a console on the client host.</p>
-
-  <p translate>Install the pre-built bootstrap RPM. This will configure the host to use this server.</p>
-
-  <pre>rpm -Uvh {{ baseURL }}/pub/{{ consumerCertRPM }}</pre>
-
-  <p translate>Use Subscription Manager to register.</p>
-
-  <pre><code>subscription-manager register --org="{{ organization.label }}" --environment="Library"</code></pre>
-
-  <p translate>Use an Activation Key to register.</p>
-
-  <pre><code>subscription-manager register --org="{{ organization.label }}" --activationkey="activationkey1"</code></pre>
-
-  <div ng-show="capsules.results.length > 0">
-    <p>
-      <h5 translate>Consuming Content From a Capsule</h5>
-      {{ "Selected Capsule:" | translate }}
-      <select ng-model="selectedCapsule"
-              ng-options="capsule.url for capsule in capsules.results">
-      </select>
-
-      <pre ng-show="selectedCapsule"><code>subscription-manager register --org="{{ organization.label }}" --baseurl="https://{{ hostname(selectedCapsule.url) }}/pulp/repos"</code></pre>
-    </p>
-  </div>
+ <p translate>To register a content host to this server, follow these steps.</p>
+  <ol>
+    <li ng-hide="noCapsulesFound">
+      <p translate>Select a Content Source:</p>
+      <p><select ng-model="selectedCapsule" ng-options="capsule.name for capsule in capsules.results"></select></p>
+    </li>
+    <li>
+      <p translate>Install the pre-built bootstrap RPM:</p>
+      <pre ng-hide="noCapsulesFound"><code>rpm -Uvh http://{{ hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+      <pre ng-show="noCapsulesFound"><code>rpm -Uvh http://{{ katelloHostname }}/pub/{{ consumerCertRPM }}</code></pre>
```

Comment:
@ehelms Something like this to handle such a case? It really shouldn't happen, and not sure if this is the best way to handle it though.